src/layout/List/config.ts

```diff
@@ -1,6 +1,13 @@
 import { CG } from 'src/codegen/CG';
 import { CompCategory } from 'src/layout/common';
 
+export const LIST_SUMMARY_OVERRIDE_PROPS = new CG.obj()
```

Comment:
This should probably be used somewhere? 🤔

Reply:
Yes!